temporian/implementation/numpy/operators/set_index.py

```diff
@@ -76,28 +75,26 @@ def _append_impl(event: NumpyEvent, append_feat_names: List[str]) -> NumpyEvent:
     }
     # initialize destination event & sampling data
     dst_event_data = {}
```

Comment:
Should we have an option to specify if the added index is added at the front of back? Alternatively, we need a function to change the order of the index values.

Reply:
I think it would make the operator's signature too overloaded. Agree on creating another op to shuffle the index levels' positions.